src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -58,6 +58,7 @@ def get_down_block(
     resnet_time_scale_shift: str = "default",
     temporal_num_attention_heads: int = 8,
     temporal_max_seq_length: int = 32,
+    temporal_double_self_attention: bool = True,
```

Comment:
We can remove this argument from here. I would import the blocks directly into the model and set the argument directly in the block.